README.rst

```diff
@@ -37,9 +37,13 @@ Current information can always be found at the NIPY website::
 Mailing Lists
 =============
 
-Please see the developer's list::
+For questions on how to use nipy or on making code contributions, please see the ``neuroimaging`` mailing list:
 
-    http://projects.scipy.org/mailman/listinfo/nipy-devel
+    https://mail.python.org/mailman/listinfo/neuroimaging
+
+For new bug reports, a list of current bugs and planned changes, or to make code contributions, please see the Github issues:
```

Comment:
What about:

    Github issues may also be used to report bugs, contribute code, or see a list of known bugs and planned changes: